### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Vidmapps**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-Share screen link to the project https://zpl.io/2Gd0wjY
+# Lint
+
+This is the project (dashboard) to learn **Front-end without** step by step guides. This is the [link](https://zpl.io/2Gd0wjY) to the project desgin work files.
```
[the remark this answers is not in the document]
Comment (alynioke) [reply README.md]: `desgin` -> `design`

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+## Technologies used (so far):
+ - **NodeJS** - platform to run JS outside the browser;
+ - **Git & GitHub** - project repository;
+ - **CanvasJS** - library to create aniamted graphs.
```
Comment (alynioke) [inline README.md]: `aniamted` -> `animated`
I think it's worth to find some plugin/setting for your text editor to help you spot typos, as i see it's not the first time it's happening. 
E.g. in phpStorm I see which words are wrong english words - they are underlined as typos by editor. 
image

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+
+ ## Packages used (so far):
```
Comment (alynioke) [inline README.md]: here and above no need to write "(so far)". Whatever is inside README implies that it's current state of the project. All document is by default "so far" :D And if anything changes - you just update readme.

File: README.md [hunk 4]
```diff
@@ -1 +1,16 @@
...
+ **None**
+
+## How it works
+1. Download the whole project from Github.
+2. Install NodeJS packages by using `npm install`.
+3. Click `index.htm` in file in browser to run in your internet browser.
```
Comment (alynioke) [inline README.md]: I see it's `html` not `htm`

**Comment by alynioke** [comment]
Writing good readme or documentation is same importance level as writing code. Once I tried to update documentation in Polaris (open source library by Shopify) and look in Discussion tab [how much comments I got](https://github.com/Shopify/polaris-react/pull/2312) about simple text 😃 
In Shopify there are even Content Strategists who check that text sound well, tone is same as in the rest of text, etc. It's not related to projects we work in Oberlo, but we still care about any docs.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vidmapps) [reply README.md]: Thanks, for noticing

File: README.md (hunk 2 above)
Comment (Vidmapps) [reply README.md]: You have a very good eye :D

**Comment by Vidmapps** [comment]
True, before I thought that devs only code, but that's only a part of their whole job role...

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (alynioke) [reply README.md]: haven't answered about *Front-end without*

**Pull request merged**